Quintor/src/main/java/com/example/quintor/DashboardController.java

```diff
@@ -63,7 +77,77 @@ public void uploadButtonAction(ActionEvent event) {
         fc.getExtensionFilters().add(new FileChooser.ExtensionFilter("MT940 files", "*.940"));
         File f = fc.showOpenDialog(null);
         if (f != null) {
+            try {
+                if (uploadFile(f, userId)) {
+                    System.out.println("gelukt");
+                } else {
+                    System.out.println("niet gelukt");
+                }
+            } catch (IOException error) {
+                error.printStackTrace();
+            }
+        }
+    }
+
+    /**
+     * This method uploads a mt940 file to the MongoDB through a POST request to the
+     * api.
+     *
+     * @param file   The file which should be uploaded.
+     * @param userId The user id of the user who uploads the file.
+     * @return return true when the file has been uploaded and false when something
+     * went wrong.
+     * @throws IOException
+     */
+    private boolean uploadFile(File file, int userId) throws IOException {
+        if (file == null || !file.isFile() || !file.exists()) {
+            return false;
+        }
+        if (userId < 0) {
+            return false;
+        }
+        String url = "http://localhost:8080/api/mt940/insert";
```

Comment:
Je kan nog een .env bestand aanmaken waar de url voor de api(http://localhost:8080). zet dit bestand dan ook in de gitignore